tuf/client_rework/metadata_wrapper.py

```diff
@@ -95,8 +95,8 @@ def keys(self, role):
         TODO
         """
         keys = []
-        for keyid in self._meta.signed.roles[role]["keyids"]:
-            key_metadata = self._meta.signed.keys[keyid]
+        for keyid in self._meta.signed.roles[role].keyids:
+            key_metadata = self._meta.signed.keys[keyid].to_dict()
```

Comment:
This is fine here but ... for future I guess we want to modify the Metadata.verify() function so that it takes a Key as argument so that this _TUF Key -> dictionary ->  SSLIB Key_ dance isn't needed ? If you agree, can you add that as comment in the already open verification API improvement issue or file a new issue about it?

Reply:
Mentioned it [here](https://github.com/theupdateframework/tuf/issues/1377#issuecomment-840444942).